strapi/strapi
# Adapt the tracking system events for both API tokens and transfer tokens

### What does it do?

Describe the technical changes you did.

### Why is it needed?

We want the events related to the tokens to be filtered by the kind of token in Amplitude, so that We can explore data regarding the transfer tokens and the api tokens.

### How to test it?

You can just check in the Network tab the API call to https://analytics.strapi.io/api/v2/track and see the different events and event labels for these events
didCreateToken
willDeleteToken
willCreateToken
didEditToken
willAccessTokenList
willEditToken
willEditTokenFromList
didCopyTokenKey
didEditTokenFromList
didAddTokenFromList
didAccessTokenList
willAddTokenFromList

For both the Transfer tokens and the API tokens

### Related issue(s)/PR(s)

The ticket related to this activity is https://strapi-inc.atlassian.net/browse/DX-617
